src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -272,6 +272,7 @@ public static class Agent
                 //
                 public static readonly string AcceptTeeEula = "agent.acceptteeeula";
                 public static readonly string AllowAllEndpoints = "agent.allowAllEndpoints"; // remove after sprint 120 or so.
+                public static readonly string AllowLegacyPowershell = "agent.allowLegacyPowershell";
```

Comment:
Do you want to add a comment on when you think this can be fully removed? maybe in 40 or so sprints ;)